draft-ietf-lake-edhoc.md

```diff
@@ -1499,17 +1501,59 @@ IANA has added the media type "application/edhoc" to the "Media Types" registry.
 
 - Change Controller: IESG
 
-## CoAP Content-Formats Registry {#content-format}
+### application/edhoc+cbor-seq Media Type Registration
 
-IANA has added the media type "application/edhoc" to the "CoAP Content-Formats" registry under the group name "Constrained RESTful Environments (CoRE) Parameters".
+- Type name: application
 
--  Media Type: application/edhoc
+- Subtype name: edhoc+cbor-seq
 
--  Encoding:
+- Required parameters: N/A
 
--  ID: TBD42
+- Optional parameters: N/A
 
--  Reference: \[\[this document\]\]
+- Encoding considerations: binary
+
+- Security considerations: See Section 7 of this document.
+
+- Interoperability considerations: N/A
+
+- Published specification: \[\[this document\]\] (this document)
+
+- Applications that use this media type: To be identified
+
+- Fragment identifier considerations: N/A
+
+- Additional information:
+
+  * Magic number(s): N/A
+
+  * File extension(s): N/A
+
+  * Macintosh file type code(s): N/A
+
+- Person & email address to contact for further information: See "Authors' Addresses" section.
+
+- Intended usage: COMMON
+
+- Restrictions on usage: N/A
+
+- Author: See "Authors' Addresses" section.
+
+- Change Controller: IESG
+
+## CoAP Content-Formats Registry {#content-format}
+
+IANA has added the media types "application/edhoc" and "application/edhoc+cbor-seq" to the "CoAP Content-Formats" registry under the group name "Constrained RESTful Environments (CoRE) Parameters".
+
+~~~~~~~~~~~
++----------------------------+----------+-------+------------------------+
+| Media Type                 | Encoding | ID    | Reference              |
++----------------------------+----------+-------+------------------------+
+| application/edhoc          | -        | TBD42 | \[\[this document\]\]  |
+| application/edhoc+cbor-seq | -        | TBD43 | \[\[this document\]\]  |
```

Comment:
Not my area of expertise. So I don't have any strong opinions.

The need seem to have a different media type for "connection ID", EDHOC

But what is the difference between "application/edhoc" and "application/edhoc+cbor-seq"? Do we need both?

- EDHOC (application/edhoc) is already always cbor-seq. 
- My understanding is that is that "edhoc+cbor-seq" means that this is edhoc but cbor-seq encoded.

My understanding would that that it would make more sense with

edhoc+cbor-seq
cid-edhoc+cbor-seq

(Feel free to ignore this comments)

Reply:
Thank you for your comment. I went for the easiest path but your suggestion totally makes sense.

I updated the PR accordingly.